app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new(author_id: current_user.id, post_id: @post)
+
+    if @like.save
+      redirect_to user_post_path(user_id: @post.author_id, id: @post.id)
+    else
+      render :new, alert: 'An error occurred'
+    end
+  end
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 it would be good to add the comments and likes counter to the `show` page of the posts so that the user would be able to see the number of likes increasing as the user clicks the button to like a post.
- your design:
![Annotation 2022-06-25 091005](https://user-images.githubusercontent.com/85113567/175764761-ecde02e9-b03d-4bf4-8b73-4fa3d50ecc87.jpg)

- Required design:
![Annotation 2022-06-25 091730](https://user-images.githubusercontent.com/85113567/175764987-350fe0f4-0868-42cc-bd78-354b68d613e5.jpg)